modules/swagger-jaxrs/src/test/java/io/swagger/BeanConfigTest.java

```diff
@@ -0,0 +1,52 @@
+package io.swagger;
+
+import static org.testng.Assert.assertNotNull;
+import static org.testng.Assert.assertEquals;
+
+import io.swagger.jaxrs.config.BeanConfig;
+import io.swagger.models.Scheme;
+import io.swagger.models.Swagger;
+
+import java.util.Arrays;
+import java.util.EnumSet;
+import java.util.HashSet;
+import java.util.Set;
+
+import org.testng.annotations.Test;
+
+public class BeanConfigTest {
+    private final Set expectedKeys = new HashSet<String>(Arrays.asList("/packageA", "/packageB"));
+    private final Set<Scheme> expectedSchemas = EnumSet.of(Scheme.HTTP, Scheme.HTTPS);
+
+    private BeanConfig createBeanConfig(String rp) {
+        BeanConfig bc = new BeanConfig();
+        bc.setResourcePackage(rp);
+        bc.setSchemes(new String[]{"http", "https"});
+        bc.setHost("petstore.swagger.io");
+        bc.setBasePath("/api");
+        bc.setTitle("Petstore Sample API");
+        bc.setDescription("A sample API that uses a petstore as an example to demonstrate features in the swagger-2.0 specification");
+        bc.setTermsOfServiceUrl("http://swagger.io/terms/");
+        bc.setContact("Swagger API Team");
+        bc.setLicense("MIT");
+        bc.setLicenseUrl("http://github.com/gruntjs/grunt/blob/master/LICENSE-MIT");
+        bc.setScan(true);
+        return bc;
+    }
+
+    @Test(testName = "scan a simple resource")
```

Comment:
The `testName` attribute will be ignored if `@Test` is not at the class level. You can use `description` instead.